sentry.rc.in

```diff
@@ -0,0 +1,22 @@
+1 VERSIONINFO
+FILEVERSION    1,0,0,0
+PRODUCTVERSION {SENTRY_VERSION_MAJOR},{SENTRY_VERSION_MINOR},{SENTRY_VERSION_PATCH},0
+BEGIN
+BLOCK "StringFileInfo"
+BEGIN
+    BLOCK "040904E4"
+    BEGIN
+        VALUE "FileDescription", "Sentry Native"
+        VALUE "FileVersion", "1.0.0.0"
+        VALUE "InternalName", "sentry-native"
+        VALUE "LegalCopyright", "https://sentry.io"
+        VALUE "OriginalFilename", "sentry.dll"
+        VALUE "ProductName", "Sentry Native"

```

Comment:
I think `ProductName` and `FileDescription` should probably be `Sentry Native SDK`.